### PR TITLE
Step 7

### PR DESCRIPTION
**Pull request by loki1520**
Hi! Game#3 - Brain-gcd!

### CODE REVIEW

File: src/games/brain-gcd.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import getRandomNumber from '../randomCount.js';
+import gamesCore from '../index.js';
+
+const gameDescription = 'Find the greatest common divisor of given numbers.';
+
+const getMaximumDivisior = (one, two) => {
```
Comment (CoraloReef) [inline src/games/brain-gcd.js]: The name of your function tells me that a number will be returned, but a string is returned. In this case, it is better to convert to a string in the generator.

File: src/games/brain-gcd.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  const minimal = one <= two ? one : two;
+  for (let i = minimal; i > 1; i -= 1) {
+    if ((two % i === 0) && (one % i === 0)) {
+      return String(i);
+    }
+  }
+  return '1';
```
Comment (CoraloReef) [inline src/games/brain-gcd.js]: Magic:)

File: src/games/brain-gcd.js (hunk 1 above)
Comment (CoraloReef) [reply src/games/brain-gcd.js]: See how recursion can make it easier to get a result
```
const getMaximumDivisior = (num1, num2) => {
  if (num2 === 0) {
    return num1;
  }
  return getMaximumDivisior(num2, num1 % num2);
};
```

Comment (loki1520) [reply src/games/brain-gcd.js]: ok, fixed!

File: src/games/brain-gcd.js (hunk 2 above)
Comment (loki1520) [reply src/games/brain-gcd.js]: )) magic disappeared)

**Review by CoraloReef — APPROVED** [review]

**Pull request merged**